plugins/catalog-backend/package-lock.json

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
accidental npm instead of yarn? - remove

Reply:
ooh yeah - idk how that made it in.